opm/io/eclipse/EclFile.hpp

```diff
@@ -118,4 +124,68 @@ class EclFile
 
 }} // namespace Opm::EclIO
 
+
+namespace Opm {
+    namespace EclIO {
+    
+    class EclFileLGR : public EclFile {
+    public:
+        EclFileLGR(const std::string& filename, bool preload = false)
+            : EclFile(filename, preload)
+        {
+            initialize_lgr_properties();
+            classify_lgr();
+        };
```

Comment:
Please don't put semi-colons on closing braces (i.e., `};`) unless it's actually needed (at the end of `struct` or `class` definition).